themes/cardinal_service/pdb_components/opportunities_list/src/Components/SelectList.jsx

```diff
@@ -149,6 +149,7 @@ export const SelectList = ({defaultValue, field, label, multiple, onChange, opti
       <Autocomplete
         classes={myStyles}
         disableCloseOnSelect
+        aria-labelledby={field+ '-label'}
```

Comment:
Instead of using `aria-label` and adding in the autcomplete's title, I was thinking that we can use `aria-labelledby` to reference the id of the autocomplete's respective label instead.

Thoughts? 🤔

Reply:
seems reasonable to me. we can try it out. did you want to test it on a dev site?

Reply:
Yes please! That way I can run AMP and see if they spit out the same errors

Reply:
https://cardinalservice-dev.sites-pro.stanford.edu/ is fair game.